index.ios.js

```diff
@@ -75,14 +72,15 @@ var WalkAbout = React.createClass({
     }
   },
 
-  toggleGeofenceState: function() {
-    this.state.inGeofence = !this.state.inGeofence;
+  backToMap: function(){
+    this.setState({inGeofence: false})
   },
 
   render: function() {
     if(this.state.inGeofence){
       console.log('rendering MonumentDetail')
-      return (<MonumentDetail monument={this.currentMonument} />)
+      return (<MonumentDetail monument={MONUMENTS[0]}
```

Comment:
Whoops, that was part of a temporary workaround for testings sake

On Tue, Mar 15, 2016 at 8:43 AM Devin Mandelbaum <notifications@github.com>
wrote:

> In index.ios.js
> <https://github.com/nyc-squirrels-2016/Walkabout/pull/10#discussion_r56155910>
> :
>
> >    },
> >
> >    render: function() {
> >      if(this.state.inGeofence){
> >        console.log('rendering MonumentDetail')
> > -      return (<MonumentDetail monument={this.currentMonument} />)
> > +      return (<MonumentDetail monument={MONUMENTS[0]}
>
> Right now the only reason I'm not merging this is the fact that it's
> passing in MONUMENTS[0] instead of this.currentMonument. I want to go over
> this, but everything else looks fantastic.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nyc-squirrels-2016/Walkabout/pull/10/files#r56155910
>